README.md

```diff
@@ -9,34 +9,27 @@ You’ll need a Kubernetes cluster to run against. You can use [KIND](https://si
 
 ### Running on the cluster
 
-1. Install Instances of Custom Resources:
-
+1. Install custom resource definitions:
 ```sh
-kubectl apply -f config/samples/
+make install
 ```
 
-2. Build and push your image to the location specified by `IMG`:
+2. Install custom resources:
 
 ```sh
-make docker-build docker-push IMG=<some-registry>/perses-operator:tag
+bin/kustomize build config/samples | kubectl apply -f -
```

Comment:
I think directly applying the config/samples dir via kubectl is not correct, the resources are supposed to be rendered by kustomize, right?

Reply:
yes, correct.